app/helpers/application_helper/toolbar/network_router_center.rb

```diff
@@ -48,8 +48,20 @@ class ApplicationHelper::Toolbar::NetworkRouterCenter < ApplicationHelper::Toolb
             :confirm      => N_('Warning: This Router and ALL of its components will be removed!'),
             :klass        => ApplicationHelper::Button::GenericFeatureButtonWithDisable,
             :options      => {:feature => :delete},
-            :data         => {'function'      => 'sendDataWithRx',
-                              'function-data' => '{"type": "delete", "controller": "toolbarActions", "payload": {"entity": "network_routers"}}'},
+            :data         => {
+              'function'      => 'sendDataWithRx',
+              'function-data' => {
+                :type       => 'delete',
+                :controller => 'toolbarActions',
+                :payload    => {
+                  :entity => 'network_routers',
+                  :labels => {
+                    :single   => 'network_router'.titleize,
+                    :multiple => 'network_router'.titleize.pluralize
```

Comment:
not quite sure about the i18n aspects of `'network_router'.titleize.pluralize`, summoning @mzazrivec :)

Reply:
`_('Network Router')` and `_('Network Routers')` please. No `.titleize`, nor `.pluralize`.